simulation-runner/src/main/scala/com/puppetlabs/gatling/runner/ConfigDrivenSimulation.scala

```diff
@@ -31,16 +32,40 @@ class ConfigDrivenSimulation extends Simulation {
     // Here we've replaced our "pause" with a Gatling "session function",
     // which basically just sets a session variable to check to see if
     // we are on the final repetition, and if not, sleep for 30 mins.
-    chain.exec((session: Session) => {
+    //
+    // In addition, simply sleeping for sleepDuration after a run causes issues
+    // long runs, where start/end times will drift, and agent runs will start to
+    // clump together. To solve this, the length of the pause is calculated to
+    // be the difference between (start time + (repetition * sleep duration))
+    // and the current time.
+    //
+    // That is, if the sleep duration is 30 seconds, and a run takes 5 seconds,
+    // the agent will sleep for 25 seconds
+    chain.exec((session) => {
```

Comment:
gotcha.  I prefer explicit in general so I'd probably add it to the other spots, but it's not a big enough deal to hold up the PR for or anything.